Vision/owlbot.py

```diff
@@ -14,34 +14,22 @@
 
 """This script is used to synthesize generated parts of this library."""
 
-import subprocess
-import synthtool as s
-import synthtool.gcp as gcp
 import logging
-
-AUTOSYNTH_MULTIPLE_COMMITS = True
+from pathlib import Path
+import synthtool as s
+import subprocess
+from synthtool.languages import php
+from synthtool import _tracked_paths
 
 logging.basicConfig(level=logging.DEBUG)
 
-gapic = gcp.GAPICBazel()
-common = gcp.CommonTemplates()
+src = Path(f"../{php.STAGING_DIR}/Vision").resolve()
+dest = Path().resolve()
 
-v1_library = gapic.php_library(
-    service='vision',
-    version='v1',
-    bazel_target=f'//google/cloud/vision/v1:google-cloud-vision-v1-php',
-)
+# Added so that we can pass copy_excludes in the owlbot_main() call
+_tracked_paths.add(src)
 
-# copy all src except partial veneer classes
-s.move(v1_library / f'src/V1/Gapic')
-s.move(v1_library / f'src/V1/resources')
-
-# copy proto files to src also
-s.move(v1_library / f'proto/src/Google/Cloud/Vision', f'src/')
-s.move(v1_library / f'tests/')
-
-# copy GPBMetadata file to metadata
-s.move(v1_library / f'proto/src/GPBMetadata/Google/Cloud/Vision', f'metadata/')
+php.owlbot_main(src=src, dest=dest, copy_excludes=["./V1/gapic_metadata.json", "./V1/ImageAnnotatorClient.php", "./V1/ProductSearchClient.php"])
```

Comment:
Need another set of eyes to confirm we should indeed exclude `gapic_metadata.json` as it didn't exist before but other product directories do have it.

Reply:
Is there a more scalable way to do this? It'll be error prone to have to manually exclude our GAPIC client classes every time. If you look at an API like Compute, they have 78 gapic clients. That would be a real pain, and also requires maintenance when clients are added.

You'll notice the way the old way does it is it only copies `Gapic` and `resource` directories from the gapic generator, as all the other files in `src` are from protobuf. Maybe we can do something with the `deep-copy`, like put them in a directory called `protos` and then move them in later? This may be a question for @bcoe

Reply:
Is the problem that `ImageAnnoltatorClient`, etc., have manual changes to them?

Could these manual changes be performed withi a post-hoc regex perhaps, rather than excluding the files?

Reply:
Yes exactly, those files are the veneer files for the clients that can be customized. I'm not sure what you mean by the Post-hoc regex, but it sounds possible for sure.

Reply:
Could we get the classes that extend a `*GapicClient` class and ignore those files? That would be handy if the number of Gapic clients is on the higher side. But is that accurate?
Not sure if that factors in @bcoe's suggestions?

Reply:
That would be super accurate. I was nervous about just excluding `*Client.php` because that's less reliable. I think matching a regex like extends `/class \w+Client extends \w+GapicClient/` would be very sound.

Reply:
@bshaffer 
Yeah I understand, but in this case, the original code just ignores all the PHP files under `src/V1`, so it won't be a problem if we all ignores the PHP files there.

I confirmed this code works:

```python
php.owlbot_main(
    src=src,
    dest=dest,
    copy_excludes=[
        src / "V1/src/V1/gapic_metadata.json",
        src / "V1/src/V1/*.php"
    ]
)

```

We can come up with a better solution if we find more complex cases.